pkg/providers/v1/aws.go

```diff
@@ -1732,6 +1777,10 @@ func (c *Cloud) InstanceTypeByProviderID(ctx context.Context, providerID string)
 		return "", err
 	}
 
+	if isFargateNode(string(instanceID)) {
+		return "", nil
```

Comment:
good point but we don't want to populate this information as Fargate abstracts the instance types. But should be easy to add our own definition for instance-type as "fargate" if we need to. But in future it might affect if Fargate has some plan to expose the task type as "compute-optimized", "memory-optimized", etc. My suggestion is to keep it simple for now and add the type as "fargate" later if needed. Do you agree as well?

Reply:
Yeah, exposing the task type would be be a good future reason to fill this in.  In that case, setting it to "fargate" shouldn't affect anything negatively, but we can keep it simple for now and wait until someone asks for something like that.